src/js/view/view.js

```diff
@@ -341,6 +338,9 @@ function View(_api, _model) {
         // Triggering 'resize' resulting in player 'ready'
         _lastWidth = _lastHeight = null;
         this.checkResized();
+        if (!this.resizeListener) {
+            this.resizeListener = new ResizeListener(_playerElement, viewsManager.scheduleResponsiveRedraw);
```

Comment:
The most optimal way to fit this into view setup would be for this classes constructor to only do the DOM creation and styling - that can happen in `view.setup`. And then defer the reads until here (or just after `updateBounds` above).